OfficeAddin/05 Using modern JavaScript/Demos/01 Office Add-in using React/README.md

```diff
@@ -1,7 +1,7 @@
 # Office Add-ins: Build an Office Add-in using modern JavaScript tools and techniques - 300 Level
 
 ----------------
-In this demo, you will build an Office Add-in using a number of popular JavaScript tools and techniques. 
+In this demo, you will build an Office Add-in using React a popular JavaScript framework. 
```

Comment:
Suggest adding a comma after the word "React" (i.e., "...using React, a popular JavaScript framework.")

Reply:
Fixed